package.json

```diff
@@ -66,12 +72,15 @@
     "loader.js": "^4.7.0",
     "prettier": "^1.18.2",
     "qunit": "^2.9.2",
+    "qunitjs": "^2.4.1",
```

Comment:
We should use `qunit` not `qunitjs` (`qunitjs` is deprecated in favor of `qunit`).